src/main/java/org/jboss/pnc/deliverablesanalyzer/Finder.java

```diff
@@ -209,7 +209,7 @@ private FinderResult find(
                 "Starting distribution analysis for {} with config {} and cache manager {}",
                 files,
                 config,
-                cacheManager != null ? cacheManager.getName() : "disabled");
+                cacheManager != null ? cacheManager : "disabled");
```

Comment:
Some reason for removing `getName()`? I don't recall what `toString()` prints.

Reply:
getName method is not present in the `BasicCacheContainer` class.

Reply:
There is `getCache().getName()` though. What does it currently print now vs. before?

Reply:
What it prints now is just the object sha. I'll try the `getCache().getName()` and report back.

Reply:
It fails with:
```
2021-08-30 15:53:32,672 WARN  [org.jbo.pnc.del.res.AnalyzeResource] (executor-thread-1) Analysis with ID 4bdcf135bf6e3a69af210994b9c61ce66b9766f16b46bf56443d38dbaa7df26d failed due to {}: org.infinispan.commons.CacheConfigurationException: ISPN000433: A default cache has been requested, but no cache has been set as default for this container
	at org.infinispan.manager.DefaultCacheManager.getCache(DefaultCacheManager.java:492)
	at org.infinispan.manager.DefaultCacheManager.getCache(DefaultCacheManager.java:138)
	at org.jboss.pnc.deliverablesanalyzer.Finder.find(Finder.java:212)
	at org.jboss.pnc.deliverablesanalyzer.Finder.lambda$find$0(Finder.java:132)
	at io.smallrye.context.impl.wrappers.SlowContextualCallable.call(SlowContextualCallable.java:21)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$13.runWith(VertxCoreRecorder.java:536)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

I guess the `getCache` call is to obtain the remote cache (whatever is default) and not what we actually want.

Reply:
I guess we don't have much choice then. I don't think what it printed before was anymore useful, might have been 'default' or something.